components/list/list-item-generic-layout.js

```diff
@@ -0,0 +1,73 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+
+class ListItemGenericLayout extends LitElement {
+
+	static get styles() {
+		return css`
+			:host {
+				display: grid;
+				grid-template-columns:
+					[start outside-control-start] minmax(0, min-content)
+					[control-start outside-control-end] minmax(0, min-content)
+					[control-end content-start] auto
+					[content-end actions-start] auto
+					[end actions-end];
+			}
+			::slotted([slot="outside-control"]),
```

Comment:
Just one thing to be aware of here, I _think_ this selector would not work in IE11.  It would need to be combined with another selector, such as `.some-class ::slotted([slot="outside-control"])`.  Not sure the root cause, but I think it's something to do with the polyfills.

Reply:
^ probably don't need to worry about this at this point, but something to be aware of.

Reply:
So, I tested this in IE11. The selector itself works because it's a direct child. However, IE11 doesn't support `grid-column` and so the grid does not work properly. I spoke to Maya about this and the plan is to only roll this out for drag 'n' drop before June, then only look at changing the current `list-item` component after we drop IE11 support.